client/my-sites/hosting/main.js

```diff
@@ -23,24 +25,34 @@ import PhpMyAdminCard from './phpmyadmin-card';
  */
 import './style.scss';
 
-const Hosting = ( { translate } ) => {
+const Hosting = ( { translate, isAtomic } ) => {
 	return (
 		<Main className="hosting is-wide-layout">
 			<PageViewTracker path="hosting/:site" title="Hosting" />
 			<DocumentHead title={ translate( 'Hosting' ) } />
 			<SidebarNavigation />
+			{ ! isAtomic && (
+				<Banner
+					title={ translate(
+						'Please active SFTP and PHPMyAdmin access to begin using these features.'
+					) }
+					icon="info"
+					callToAction={ translate( 'Activate' ) }
+					disableHref
+				/>
+			) }
 			<div className="hosting__cards">
-				<SFTPCard />
-				<PhpMyAdminCard />
+				<SFTPCard disabled={ ! isAtomic } />
+				<PhpMyAdminCard disabled={ ! isAtomic } />
 			</div>
 		</Main>
 	);
 };
 
 export default connect( state => {
+	const siteId = getSelectedSiteId( state );
+
 	return {
-		site: getSelectedSite( state ),
-		siteId: getSelectedSiteId( state ),
-		siteSlug: getSelectedSiteSlug( state ),
+		isAtomic: isSiteAutomatedTransfer( state, siteId ),
```

Comment:
We can make this a little easier to read with something like:
```jsx
return {
    isDisabled: ! isSiteAutomatedTransfer( state, siteId ),
```

If the variable name is too generic folks are always welcome to be more verbose.